lib/twilio/audiohelper.ts

```diff
@@ -275,6 +275,16 @@ class AudioHelper extends EventEmitter {
     if (isEnumerationSupported) {
       this._initializeEnumeration();
     }
+
```

Comment:
Let's add a comment here about firefox not working and add a link to the firefox issue.